invenio_app_ils/circulation/api.py

```diff
@@ -192,6 +193,19 @@ def checkout_loan(
         circulate.
     """
     loan_cls = current_circulation.loan_record_cls
+    # In case of self checkout confirm:
+    #   1. self checkout feature flag is enabled
+    #   2. The patron ids match
+    if (
+        kwargs.get("self_checkout", False)
```

Comment:
just a reminder: we agreed to check if the action is performed by the librarian, not to pass the kwarg from the UI, since it is more reliable check.